config/config.yml

```diff
@@ -29,10 +29,11 @@ styleguideKitPath: vendor/pattern-lab/styleguidekit-twig-default
 lineageMatch: '{%([ ]+)?(?:include|extends|embed)( |\()[&quot;\&#039;]([\/.@A-Za-z0-9-_]+)[&quot;\&#039;]([\s\S+]*?)%}'
 lineageMatchKey: 3
 patternExtension: twig
-twigDebug: false
+twigDebug: true
 twigAutoescape: false
 twigDefaultDateFormat: ""
 twigDefaultIntervalFormat: ""
+twigExtensions: - '\Ajgl\Twig\Extension\BreakpointExtension'
```

Comment:
Husky is doing this, but this invalid and causes an error. It needs to be:

```yml
twigExtensions:
  - '\Ajgl\Twig\Extension\BreakpointExtension'
```

Reply:
Yep, this works. Thank you!

Maybe there are some rules we can tweak?  It's currently changing the quoting to `"` which is also annoying when you have HTML inside and need to single quote the attributes.